src/pydocstyle/violations.py

```diff
@@ -202,7 +202,9 @@ def to_rst(cls):
 D400 = D4xx.create_error('D400', 'First line should end with a period',
                          'not {0!r}')
 D401 = D4xx.create_error('D401', 'First line should be in imperative mood',
-                         '{0!r}, not {1!r}')
+                         "'{0}', not '{1}'")
+D401b = D4xx.create_error('D401', 'First line should be in imperative mood; '
```

Comment:
I did not. The problem here is that on Python 2, unicode strings have a repr like `u'foo'` which is both unsightly and causes test failures.